portal/models/qb_timeline.py

```diff
@@ -370,7 +414,7 @@ def update_users_QBT(user_id, invalidate_existing=False):
 
         # As we move forward, capture state at each time point
 
-        pending_qbts = []
+        pending_qbts = AtOrderedList()
```

Comment:
Why not just define a sorted "view" (function) of the list (eg `.keys()`, `.items()`, `.values()`)?

Reply:
tried to make this clear in the comment - we need to preserve insertion order AND then by `at`.  Using a built in sorted view won't retain this need.